files/build_templates/docker_image_ctl.j2

```diff
@@ -7,18 +7,32 @@ HWSKU=`sonic-cfggen -m /etc/sonic/minigraph.xml -v minigraph_hwsku`
 start() {
     docker inspect --type container {{docker_container_name}} &>/dev/null
     if [ "$?" -eq "0" ]; then
+{%- if docker_container_name == "database" %}
```

Comment:
why do you want to treat database differently? if database is hwsku independent, then we should not mount the hwsku directory into database.

Reply:
Taoyu has a change to remove that mounting. He said he will take care of that change after my change went in.

Reply:
but, you do not really need to treat database docker differently, right?

Reply:
Please see the conversation in the next comments.

Thanks,
Ying